core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -23,11 +23,17 @@
 import java.util.Map;
 import java.util.Set;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;
+import org.apache.iceberg.view.ViewMetadata;
+import org.apache.iceberg.view.ViewVersion;
 
-/** Represents a change to table metadata. */
+/** Represents a change to table/view metadata. */
```

Comment:
What about "table or view"? I'm not sure we're gaining much by using "/" instead.